drivers/hmis/app/models/hmis/form/definition.rb

```diff
@@ -31,6 +31,16 @@ class Hmis::Form::Definition < ::GrdaWarehouseBase
     PROJECT_COC: 'Project CoC',
   }.freeze
 
+  FORM_ROLE_CONFIG = {
+    SERVICE: { class_name: 'Hmis::Hud::HmisService', permission: :can_edit_enrollments, resolve_as: 'Types::HmisSchema::Client' },
```

Comment:
Why do services resolve_as clients?  Everything else seems to resolve as itself.

Reply:
Copy-paste error, thanks!